vite.config.ts

```diff
@@ -0,0 +1,27 @@
+import { defineConfig } from "vite";
+
+import dotenv from "dotenv";
+dotenv.config();
+
+export default defineConfig({
+  define: {
+    "import.meta.env.VITE_TREE_SITTER_FILES_S3_URL": JSON.stringify(
+      process.env.VITE_TREE_SITTER_FILES_S3_URL,
+    ),
+  },
+  build: {
+    lib: {
+      entry: "src/index.ts",
+      name: "MyLibrary",
```

Comment:
Is this name intentional?? and the fileName also?

Reply:
updated